types/jest-image-snapshot/jest-image-snapshot-tests.ts

```diff
@@ -60,7 +60,7 @@ it('Should be able to use callback as customSnapshotIdentifier', () => {
 });
 
 it('mutates original state', () => {
-    const originalState = { some: 'value' };
+    const originalState: { some: string, another?: string } = { some: 'value' };
```

Comment:
I think this is a prime example why you cannot be changing toEqual.

Reply:
I've fixed the types so that this is no longer necessary.